src/golang/lib/dynamic/engine.go

```diff
@@ -30,9 +32,12 @@ const (
 	K8sClusterUpdateAction k8sClusterActionType = "update"
 )
 
-const stateLockErrMsg = "Error acquiring the state lock"
+const (
+	stateLockErrMsg      = "Error acquiring the state lock"
+	K8sClusterNameSuffix = "aqueduct_ondemand_k8s"
```

Comment:
This is the name of the K8s integration right? Not the eks cluster name.

Reply:
yeah, updated to `K8sIntegrationNameSuffix`